dockerfiles/podman/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2019 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+FROM quay.io/podman/stable:v1.5.0
+
+RUN dnf -y install mc
+
+RUN adduser user
```

Comment:
I can't do it because I need to set permissions on '/etc/passwd' and '/home' to allow arbitrary users to write
https://github.com/eclipse/che-theia/pull/502/files#diff-065449de107c094dfd16627bead73bcaR20